website_and_docs/content/documentation/webdriver/browsers/chrome.en.md

```diff
@@ -373,7 +373,7 @@ You can drive Chrome Cast devices, including sharing tabs
 {{< badge-code >}}
 {{< /tab >}}
 {{% tab header="Python" %}}
```

Comment:
Am still going to test locally with the github code blocks. Will close this comment when done.